joomla/joomla-cms
# Wrong message after publishing nested sub-items

This PR to replace https://github.com/joomla/joomla-cms/pull/8038 where nested is not taken into account.
How to test:

```
Navigate to Category Manager
Create a main category
Create a sub-category that is related to the new main category
Unpublish the main category
Both main & sub-category will get unpublished
Click on the sub-category and try to publish and check the message
```

Do the same for sub_menus and sub_tags
